test/export/note_sorter_spec.py

```diff
@@ -0,0 +1,122 @@
+from mamba import describe, it, context, before
+from unittest.mock import MagicMock
+from random import shuffle
+
+from test_utils.anki import mock_anki_modules
+
+mock_anki_modules()
+
+from crowd_anki.export.note_sorter import NoteSorter
+from crowd_anki.config.config_settings import ConfigSettings
+
+
+test_guids = ["aaabbbccc", "bbbcccddd", "cccdddeee", "dddeeefff", "eeefffggg", "fffggghhh"]
+test_flags = [0, 1, 2, 3, 4, 5]
+test_tags = ["adjectives", "directions", "interesting", "nouns", "verbs", "zzzzFinal"]
+test_notemodels = ["Default", "LL Noun", "LL Sentence", "LL Verb", "LL Word", "Zulu"]
+test_notemodelids = test_guids
+test_fields = test_tags
+
+
+class NoteSorterTester():
+    def __init__(self):
+        self.notes = self.setup_notes()
+
+        self.config = ConfigSettings(False)
+        self.config.export_deck_sort_reversed = False
+
+    def get_single_note_mock(self, i):
+        note = MagicMock()
+
+        note.anki_object.guid = test_guids[i]
```

Comment:
assign `note.anki_object` to a variable? or initialize it via dict maybe

Reply:
I thought the MagicMock took care of that. It doesn't seem to mind :sweat_smile:

Reply:
sure it does not mind, I though it may be cleaner that way though ;)